pyproject.toml

```diff
@@ -63,7 +63,7 @@ flake8-bugbear = "^22.9"
 flake8-debugger = "^4.0"
 flake8-isort = "^4.0"
 flake8-eradicate = "^1.0"
-flake8-bandit = ">=2.1,<4"
+flake8-bandit = ">=2.1,<5"
```

Comment:
Can we bump the lower part?

Reply:
@sobolevn Yes. Which lower bound do you want?   `>=3` ?

Reply:
I prefer to use the exact `^` pin. So is going to be `^4.whatever-it-is`

Reply:
@sobolevn Done. Is this OK now?